src/app/Authenticated.tsx

```diff
@@ -8,52 +8,39 @@ import Error from '../pages/Error';
 const Admin = lazy(() => import('../pages/Admin'));
 
 const Captures = lazy(() => import('../pages/Captures'));
-const CaptureCreation = lazy(() => import('components/capture/creation/index'));
+const CaptureCreate = lazy(() => import('components/capture/create/index'));
 
 const Materializations = lazy(() => import('../pages/Materializations'));
-const NewMaterialization = lazy(
-    () => import('../components/materialization/creation')
+const MaterializationCreate = lazy(
+    () => import('../components/materialization/create')
 );
 
 const Collections = lazy(() => import('../pages/Collections'));
 
-const Builds = lazy(() => import('../pages/Builds'));
-
-const Users = lazy(() => import('../pages/Users'));
-
-const Alerts = lazy(() => import('../pages/Alerts'));
-
-const Logs = lazy(() => import('../pages/Logs'));
-
 const Authenticated = () => {
     return (
         <Suspense fallback={<Skeleton animation="wave" />}>
             <Routes>
                 <Route element={<AppLayout />}>
                     <Route path="/" element={<Home />} />
                     <Route path="/dashboard" element={<Home />} />
-                    <Route path="/app">
```

Comment:
Go rid of the `app` wrapper. Initially with how auth was handled this was useful but now it adds nothing.